dist/filelist.bzl

```diff
@@ -106,6 +105,7 @@ WT_FILELIST = ['src/block/block_addr.c',
  'src/history/hs_cursor.c',
  'src/history/hs_rec.c',
  'src/history/hs_verify.c',
+ 'src/log/cur_log.c',
```

Comment:
I think this should be renamed to `log/log_cursor.c`. That makes it consistent with `btree/bt_cursor.c`, `lsm/lsm_cursor.c` and `history/hs_cursor.c`.

Reply:
Yes, will make the change.